CHANGELOG.md

```diff
@@ -77,6 +77,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * [#436](https://github.com/allora-network/allora-chain/pull/436/files) Fix bug related to excessive usage of topic ground truth lag and misleading error message
 * [#544](https://github.com/allora-network/allora-chain/pull/544) Added check against zero-rewards after conversion to cosmosInt
 * Additional bugfixes and improvements
+* [#547](https://github.com/allora-network/allora-chain/pull/547) Improve error handling on InsertPayload, fixed/added tests err handling
```

Comment:
we should merge #546 so that this is not overwriting the wrong changelog version

Reply:
546 has been merged this should be moved up above to v0.4.0 it is updating the wrong changelog

Reply:
@relyt29 is correct

Reply:
Moved to 0.4.0 / Fixed. 
I think it would be good to have these fixes in the next upgrade. 

However, What marks the boundary between 0.3.0 and 0.4.0 now? Are all of these changes added to the 0.3.0 changelog part of the already-released 0.3.0?